lib/bolt/target.rb

```diff
@@ -77,8 +89,17 @@ def features
       end
     end
 
+    # TODO: WHAT does equality mean here?
+    # should we just compare names? is there something else that is meaninful?
```

Comment:
What you've got here makes sense to me.

Reply:
Yeah I think this is a question to answer later.